docs/installation-and-operations/operation/restoring/README.md

```diff
@@ -121,21 +121,18 @@ For Docker-based installations, assuming you have a backup as per the procedure
 
 Let's assume you want to restore a database dump given in a file, say `openproject.sql`.
 
+This assumes that the database container is called `db_1`. Find out the actual name on your host using `docker ps | postgres`.
+
 If you are using docker-compose this is what you do after you started everything for the first time using `docker-compose up -d`:
 
 1. Stop the OpenProject container using `docker-compose stop web worker`.
 2. Drop the existing, seeded database using `docker exec -it db_1 psql -U postgres -c 'drop database openproject;'`
-3. Recreate the database using `docker exec -it db_1 psql -U postgres -c 'create database openproject owner openproject;'`<sup>*</sup>
-4. Copy the dump onto the container: `docker cp openproject.sql db_1:/`
-5. Source the dump with psql on the container: `docker exec -it db_1 psql -U postgres` followed first by `\c openproject` and then by `\i openproject.sql`. You can leave this console by entering `\q` once it's done.
-6. Delete the dump on the container: `docker exec -it db_1 rm openproject.sql`
-7. Restart the web and worker processes: `docker-compose start web worker`
-
-_\* If your database doesn't have an OpenProject user you either have to create one or use which ever user you used before._
-
-* * *
-
-This assumes that the database container is called `db_1`. Find out the actual name on your host using `docker ps | postgres`.
+3. Create a user/role for the OpenProject database: `docker exec -it db_1 psql -U postgres -c 'create user openproject;'`
```

Comment:
```suggestion
3. If your database doesn't have an openproject user yet, create it with this command: `docker exec -it db_1 psql -U postgres -c 'create user openproject;'`
```

As the user might already be present, we should keep the comment from earlier

Reply:
Just an off topic question, I followed this docs that I wanted to edit in this PR: I installed OP via docker-compose and after up -d and down web worker, I dropped the database OpenProject, the user OpenProject was not there afterwards, before I don't know.